homeassistant/components/lifx/migration.py

```diff
@@ -2,14 +2,15 @@
 from __future__ import annotations
 
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.core import HomeAssistant
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers import device_registry as dr, entity_registry as er
 
 from .const import _LOGGER, DOMAIN
 from .discovery import async_init_discovery_flow
 
 
-async def async_migrate_legacy_entries(
+@callback
+def async_migrate_legacy_entries(
```

Comment:
We can do the same for `async_migrate_entities_devices` below.

Reply:
Will adjust if wifi works on the ✈️, otherwise tomorrow since ~14 hours of travel ahead